cpp/src/barretenberg/honk/composer/ultra_honk_composer.test.cpp

```diff
@@ -39,14 +39,19 @@ std::vector<uint32_t> add_variables(auto& composer, std::vector<fr> variables)
  * @param honk_prover
  * @param plonk_prover
  */
+// NOTE: Currently only checking witness polynomials (wires, sorted lists) and table polys. The permutation polys
+// are computed differently between plonk and honk and we do not enforce non-zero selectors in Honk so the final
+// element in the selectors will disagree.
 void verify_consistency(honk::UltraProver& honk_prover, plonk::UltraProver& plonk_prover)
 {
     // Check that all lagrange polys agree
     auto& honk_store = honk_prover.key->polynomial_store;
     auto& plonk_store = plonk_prover.key->polynomial_store;
     for (auto& entry : honk_store) {
         std::string key = entry.first;
-        if (plonk_store.contains(key)) {
+        bool is_sorted_table = (key.find("s_") != std::string::npos);
+        bool is_table = (key.find("table_value_") != std::string::npos);
+        if (plonk_store.contains(key) && (is_sorted_table || is_table)) {
```

Comment:
The wires are checked just below this. It has to be done in a separate loop since the wires are owned directly by the prover in Honk but they are still stored in the procing_key for Plonk. I've added a comment to clarify.